test/document-to-html.tests.js

```diff
@@ -436,6 +436,28 @@ test('header rows are wrapped in thead', function() {
     });
 });
 
+test('First column cells are <th> tags', function() {
```

Comment:
I think the test case here is really that cells marked as `isTableHeader` are turned into `<th>`s -- the context shouldn't affect what gets generated. Perhaps this test case could be simplified to make that clear?